### PR TITLE
Added the links in the table response now

### LINKED ISSUE #1360: Identify and display links in the table responses
**Actual Behaviour**

In a table response there are link responses and they are displayed as a simple text.

**Expected Behaviour**

Links should be identified and displayed as links

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by batbrain7**
Fixes #1360

Changes: Added the links in the table action type responses.

Screenshots for the change: 
screenshot_20180610-082819

### CODE REVIEW

**Comment by batbrain7** [comment]
@codedsun @imjog please review!

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/VerticalRecyclerAdapter.java [hunk 1]
```diff
@@ -2,14 +2,21 @@
...
 import android.support.annotation.NonNull;
 import android.support.v7.widget.RecyclerView;
+import android.util.Patterns;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+
 import org.fossasia.susi.ai.R;
 import org.fossasia.susi.ai.chat.adapters.viewholders.VerticalCellViewHolder;
+
```
Comment (codedsun) [inline app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/VerticalRecyclerAdapter.java]: Is this extra line needed?

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/VerticalRecyclerAdapter.java [hunk 2]
```diff
@@ -2,14 +2,21 @@
 
 import android.support.annotation.NonNull;
 import android.support.v7.widget.RecyclerView;
+import android.util.Patterns;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+
```
Comment (codedsun) [inline app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/VerticalRecyclerAdapter.java]: Extra line!

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/VerticalRecyclerAdapter.java (hunk 1 above)
Comment (batbrain7) [reply app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/VerticalRecyclerAdapter.java]: it separates the type of dependencies that they are viewed better

File: app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/VerticalRecyclerAdapter.java (hunk 2 above)
Comment (batbrain7) [reply app/src/main/java/org/fossasia/susi/ai/chat/adapters/recycleradapters/VerticalRecyclerAdapter.java]: same as above

**Comment by batbrain7** [comment]
@codedsun the click listener in separate issue

**Comment by iamareebjamal** [comment]
The commit message should be informative

https://chris.beams.io/posts/git-commit/

**Comment by batbrain7** [comment]
@iamareebjamal will remember from now

File: app/src/main/res/layout/table_recylcer_item.xml [hunk 3]
```diff
@@ -16,7 +16,15 @@
...
         android:layout_height="wrap_content"
         android:padding="5dp"
+        android:visibility="gone"
         android:id="@+id/data_item"
         />
-
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:padding="5dp"
```
Comment (arundhati24) [inline app/src/main/res/layout/table_recylcer_item.xml]: Use dimen resource. Also, please change the padding to a standard value. Can 4dp be used? :)

Comment (batbrain7) [reply app/src/main/res/layout/table_recylcer_item.xml]: doing the changes

**Comment by batbrain7** [comment]
@arundhati24 please check!

**Pull request merged**